Lib/unittest/mock.py

```diff
@@ -1620,8 +1620,6 @@ class _patch_dict(object):
     """
 
     def __init__(self, in_dict, values=(), clear=False, **kwargs):
-        if isinstance(in_dict, str):
-            in_dict = _importer(in_dict)
```

Comment:
Is the setting of `self.in_dict` on the line below still needed?

Reply:
I think it's needed since it's self.dict is passed in https://github.com/python/cpython/pull/12000/files/f2abf2667ca660f11db62bb4038529ada8d52925#diff-ff75b1b83c21770847ade91fa5bb2525L1652 and other places in the class.